keyResolver.js

```diff
@@ -1,19 +1,32 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
+import {httpClient} from '@digitalbazaar/http-client';
+import assert from 'assert-plus';
+import didIo from 'did-io';
+import {driver as keyDriver} from 'did-method-key';
+import {driver as veresDriver} from 'did-veres-one';
 
-import axios from 'axios';
+// config did-io to support did:key and did:v1 drivers
+didIo.use('key', keyDriver());
+didIo.use('v1', veresDriver());
 
-// FIXME: make more restrictive, support `did:key` and `did:v1`
-// TODO: could be made more restrictive is based on a config option that
-//       specifies where the KMS is.
 async function keyResolver({id} = {}) {
-  const headers = {Accept: 'application/ld+json, application/json'};
-  const response = await axios.get(id, {
-    headers
-  });
-  return response.data;
+  assert.string(id, 'id');
+  if(id.startsWith('did:')) {
+    return didIo.get({did: id, forceConstruct: true});
+  }
+  const origin = `${location.protocol}//${location.hostname}`;
+  if(id.startsWith(origin)) {
```

Comment:
```suggestion
  if(id.startsWith(location.origin)) {
```

Reply:
@dlongley some tests failed when I used `location.origin`. `https://localhost:9876` is what gets logged for `location.origin` and id from the tests has `https://localhost:18443`. So, in order to make the tests pass I removed the port number.

Reply:
@JSAssassin -- We can't remove the port number, however -- it changes the origin. We'll need to figure out a way to adjust the tests instead.